include/linux/soundwire/sdw.h

```diff
@@ -252,6 +254,8 @@ struct sdw_dp0_prop {
 	bool simple_ch_prep_sm;
 	u32 ch_prep_timeout;
 	bool imp_def_interrupts;
+	int num_lanes;
```

Comment:
Same, basically the rule of thumb is to run `pahole` each time after updating the structure (data type) and see if it can be improved.

Reply:
I don't think anyone has ever used this tool on the SOF driver. I just tried for 15mn without any success, and it's just yet another possibly great tool that I'll never remember come next next. Oh well.

Reply:
> I don't think anyone has ever used this tool on the SOF driver. I just tried for 15mn without any success, and it's just yet another possibly great tool that I'll never remember come next week. Oh well.

Should be easy to use...

Reply:
we didn't have the right Kconfig options, works fine now

https://github.com/thesofproject/kconfig/pull/98

Reply:
done